examples/federated_learning/yolov5_coco128_mistnet/interface.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sedna.algorithms.aggregation import MistNet
+from sedna.algorithms.client_choose import SimpleClientChoose
+from sedna.common.config import Context
+from sedna.core.federated_learning import FederatedLearning
+
+simple_chooser = SimpleClientChoose(per_round=1)
+
+# It has been determined that mistnet is required here.
+mistnet = MistNet(cut_layer=Context.get_parameters("cut_layer"),
+                  epsilon=Context.get_parameters("epsilon"))
+
+# The function `get_transmitter_from_config()` returns an object instance.
+s3_transmitter = FederatedLearning.get_transmitter_from_config()
+
+
+class Dataset:
+    def __init__(self) -> None:
+        self.parameters = {
+            "datasource": "YOLO",
+            "data_params": "./coco128.yaml",
+            # Where the dataset is located
+            "data_path": "./data/COCO",
+            "train_path": "./data/COCO/coco128/images/train2017/",
+            "test_path": "./data/COCO/coco128/images/train2017/",
+            # number of training examples
+            "num_train_examples": 128,
+            # number of testing examples
+            "num_test_examples": 128,
+            # number of classes
+            "num_classes": 80,
+            # image size
+            "image_size": 640,
+            "classes":
```

Comment:
These classes names are same with `names` of config `examples/federated_learning/yolov5_coco128_mistnet/coco128.yaml`, can we reuse that?

Reply:
This belongs to the user file (coco128 for user dataset specification). We suggest keeping it